src/lib/abacus/websocket.ts

```diff
@@ -28,7 +28,7 @@ class AbacusWebsocket implements Omit<AbacusWebsocketProtocol, '__doNotUseOrImpl
 
   private isConnecting: boolean = false;
 
-  orderbookCandlesToggleOn: boolean = true;
+  orderbookCandlesToggleOn: boolean = false;
```

Comment:
this line is fine and can stay this way even after we remove the flag/set it to true since this [line](https://github.com/dydxprotocol/v4-web/blob/main/src/hooks/tradingView/useOrderbookCandles.ts#L34) handles syncing it with FE state on mount